.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,35 +1,43 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: Bug Report
+about: Spotted a bug? Report it to us!
 title: ''
 labels: 'BUG'
 assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+**Bug Description**
+<!-- A clear and concise description of what the bug is -->
+
 
 **To Reproduce**
-Please provide a minimal working example. For instance:
+<!-- Please provide a minimal working example. For instance: -->
 
+Code -  
 ```python
 from qutip import identity
 print(identity(2))
 ```
-The terminal out put is
+
+Output -  
 ```
 Quantum object: dims = [[2], [2]], shape = (2, 2), type = oper, isherm = True
 Qobj data =
 [[1. 0.]
  [0. 1.]]
 ```
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+
+**Expected Behavior**
+<!-- A clear and concise description of what you expected to happen -->
+
 
 **Your Environment**
-Please use `qutip.about()` to get the information about your environment and paste it here.
+<!-- Please use `qutip.about()` to get the information about your environment and paste it here. -->
+Output of `qutip.about()` -  
```

Comment:
@Chasmiccoder Thank you for the PR. LGTM!

Could you also add the following to the "your environment" section? Sometimes people also include the line `===========` of the output, which will be wrongly interpreted as headers of markdown.

Output of `qutip.about()` -
```
QuTiP Version:      ***
Numpy Version:      ***
Scipy Version:      ***
Cython Version:     ***
Matplotlib Version: ***
Python Version:     ***
Number of CPUs:     ***
BLAS Info:          ***
OPENMP Installed:   ***
INTEL MKL Ext:      ***
Platform Info:      ***
```

This should make it clearer that these are the only information needed.

Reply:
Whoa, totally missed that.
Thanks!